wp-content/themes/engage/src/Models/Homepage.php

```diff
@@ -99,7 +99,6 @@ public function queryPosts($is_featured, $verticalName){
         $args = [
             'postType' => 'research',
             'vertical' => $verticalName,
-            'postsPerPage' => '100',
```

Comment:
Rather than remove this, let's set it to the number that we actually want. How many posts do we need in each context? You can add this as an additional param with a default of 10, if that makes the most sense.